<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSQRCodeLoginForm</class>
 <widget class="QWidget" name="GBSQRCodeLoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>840</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgBanner">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>20</y>
     <width>345</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblNoAccount">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>750</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
font-size: 14px; 
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="btnLoginBiz" native="true">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>204</y>
     <width>501</width>
     <height>548</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>469</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScanQRodeLogin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>133</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
		QPushButton:pressed {
		  background-color: #9C1F38;
		  padding-left: 1px; 
		  padding-top: 1px; 
		}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAuthorizeCodeLogin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>133</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoginGBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>500</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 1px solid #E9EBED;
color:#FFFFFF;
border-radius: 24px;
padding: 5px;
background: #1B2846;
font-size: 32px;
text-align: center;
}
QPushButton:pressed{
background-color: #1B2846;
 padding-ringht: 3px;
padding-bottom: 3px;
}

</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblQRCode">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>180</y>
      <width>245</width>
      <height>245</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblOpenApp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>429</y>
      <width>255</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblConstraintAndPrivacy">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>840</y>
     <width>240</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
